tlatools/org.lamport.tlatools/src/tlc2/output/MP.java

```diff
@@ -498,6 +499,9 @@ private static String getMessage0(int messageClass, int messageCode, String[] pa
             break;
         case EC.TLC_INVARIANT_VIOLATED_INITIAL:
             b.append("Invariant %1% is violated by the initial state:\n%2%");
+            if (parameters.length > 2) {
+                b.append("\nMore information: %3%");
```

Comment:
The prefix "More information: " (used here and below) doesn’t provide much value to readers. How can we make deserialization easier for downstream tools like our VSCode extension?  Can we avoid regex-based parsing, or make it dead simple?

Reply:
The answer is probably to implement a LSP API instead of investing in parsing the text output of the tool.

Reply:
Ideally, TLC would be fully machine-readable (and controllable).  Better yet, in a way that eliminates the need for parsing altogether (think proper IPC). Unfortunately, I believe this is beyond the scope if this issue.

Reply:
It is an obscure term, and I'm open to suggestions.

Maybe "Values of relevant names" would be a bit friendlier?

Reply:
I kinda like the term. It was immediately obvious what it meant from the PR title, and I've struggled to find any better term (using the clunky "bound identifiers") in the build-your-own-TLA-tools tutorial.